objects/obj_pause/Draw_0.gml

```diff
@@ -15,4 +15,25 @@ if (global.pause and !global.gameover) {
 	
 	// set back to default
 	draw_set_color(c_black);
+	
+	// #########################
+	// ### Draw menu options ###
+	// #########################
+	
+	// draw arrow
+	arrowGap = 50;
+	draw_sprite(spArrowDrumsticks, image_index, _x - arrowGap, _y[option]);
+
+	// draw text
+	draw_set_font(titleFont);
```

Comment:
Can we draw this font in a more contrasting color? At least white, possibly with a darker background.